.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: '${{ matrix.os }}'
     strategy:
       matrix:
-        os: [ ubuntu-18.04 ]
+        os: [ ubuntu-18.04, windows-2019 ]
         java-version: [ 8 ]
         python-version: [ '2.7', '3.4', '3.5', '3.6', '3.7', '3.8', '3.9', '3.10' ]
```

Comment:
Let's just remove 3.4 in the CI for now. It's EOL 3 years ago (https://www.python.org/downloads/release/python-340/)

Reply:
Yea